elementary/monitor/alerts/schema/alert.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+
+class AlertSuppressionSchema(BaseModel):
+    suppression_status: str = None
```

Comment:
Could you maybe add an `Optional` typing?
I want us to start using [`mypy`](https://github.com/python/mypy) soon as it really helps with typing.

Reply:
Sure!